### ISSUE TITLE
athtech.slack.com

### ISSUE THREAD

**Issue by Zorbash** [athtech/athtech.github.io#4]
Is athtech.slack.com affiliated with the site?
Do we have some sort of slack invitation automation?
If not we could use https://github.com/rauchg/slackin

**Comment by dark5un** [comment]
athtechorg.slack.com has been created and you have been invited hooray!!!

**Comment by leoapost** [comment]
Sorry for my late response but I had my notifications disabled and I just saw this issue.

I created [athtech](https://athtech.slack.com) slack team back in 2015 to reserve the name but I was very busy to spread the word. 

I propose to keep `athtech` instead `athtechorg` since it is easier to be discovered and we keep consistency with the xxxtech name convention (e.g. skgtech).

@Zorbash I have seen your slackin PR but I suggest to hold it a bit longer till we decide which team name we will keep.

**Comment by Zorbash** [comment]
Ι'm too in favor of keeping the convention of `xxxtech`. It's easy to update #5 if we decide to use `athtech.slack.com`. @dark5un what do you say?

**Comment by dark5un** [comment]
Yes I fine with what @leoapost says... Use the original one pls. And close this issue.

**Comment by leoapost** [comment]
Great, thank you @dark5un.

@Zorbash please, go ahead and update #5 when you find some free time and I will merge it.

**Comment by Zorbash** [comment]
:tada: Please mind inviting me to athtech.slack.com to update #5.
